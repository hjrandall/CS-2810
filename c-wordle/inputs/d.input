[g]u(e)s[s]
