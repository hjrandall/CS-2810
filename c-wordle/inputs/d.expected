517: geres
507: gares
507: genes
505: gates
502: gales
487: gades
487: games
486: gapes
486: gazes
485: gages
481: gyres
479: gytes
476: gores
475: gites
471: goles
465: geats
465: gybes
465: gyves
461: gibes
461: gives
